src/pages/Subreddit.js

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect, useState} from 'react';
+import {useHistory, useParams} from "react-router-dom";
+import axios from "axios";
+
+function Subreddit() {
+    const [posts, setPosts] = useState('');
```

Comment:
Als hier een object uiteindelijk in komt te staan dan zou ik als default waarde `undefined` of `{}` pakken, omdat het per definitie geen string zal zijn

Reply:
Bedankt voor je feedback Zwen, fijne feestdagen!